test/runtests.jl

```diff
@@ -61,9 +61,23 @@ end
                 @test neighbors(Y) == k
                 @test length(vertices(Y)) > 1
             end
+            
+            # test if we provide pre-computed Gram matrix
+            if algorithm == DiffMap && T == Float64
+                kernel = (x, y) -> exp(-sum((x .- y) .^ 2) / 1.0) # default kernel
```

Comment:
Drop `/ 1.0`  to get `Float32` kernel element type.

Reply:
ah yes. done.